test/javascripts/acceptance/manage-test.js

```diff
@@ -88,4 +88,15 @@ acceptance("Team Building: Manage", function (needs) {
       "New Name"
     );
   });
+
+  test("has links in sidebar", async (assert) => {
+    await visit("/team-build/manage");
+
+    await click(".sidebar-more-section-links-details-summary");
+
+    assert.strictEqual(
+      query(".sidebar-section-link[data-link-name='team-building']").innerText,
+      "Team Building"
+    );
```

Comment:
Spreading the good word of qunit-dom 😌 😄 

```suggestion
    assert
      .dom(".sidebar-section-link[data-link-name='team-building']")
      .hasText("Team Building");
```

Reply:
Yeah this is great man